smartgymapi/models/user_activity.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+import uuid
+
+from smartgymapi.models.meta import Base, DBSession as session, LineageBase
+
+from sqlalchemy import Column, Date, DateTime, ForeignKey, cast
```

Comment:
> Imports should be grouped in the following order:
standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports